packages/jest-preset/jest-preset.json

```diff
@@ -0,0 +1,13 @@
+{
+  "moduleNameMapper": {
+    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/node_modules/jest-preset-hops/fileMock.js",
```

Comment:
I think we ought to support the same file types here and in `url`/`file` loaders. That would mean these:

`(png|gif|jpe?g|webp|html|svg|((o|t)tf)|woff2?|ico)`